src/Og.php

```diff
@@ -205,6 +212,54 @@ public static function isGroupAudienceField(FieldDefinitionInterface $field_conf
   }
 
   /**
+   * Get the name of the group-audience type field.
+   *
+   * @param $entity_type
+   *   The entity type.
+   * @param $bundle_name
+   *   The bundle name to be checked.
+   * @param $group_type
+   *   Filter list to only include fields referencing a specific group type.
+   * @param $group_bundle
+   *   Filter list to only include fields referencing a specific group bundle.
+   *   Fields that do not specify any bundle restrictions at all are also
+   *   included.
+   *
+   * @return
+   *   Array keyed with the field name and the field label as the value.
+   */
+  public static function getGroupAudienceFields($entity_type, $bundle_name, $group_type = NULL, $group_bundle = NULL) {
```

Comment:
@pfrenssen you can indeed have multiple group audience fields. This usually makes more sense when thinking of the user entity. For example `simple_ref` would reference a group using the default OG membership type; and `premium_ref` would reference using some Premium OG membership type.

Each membership created also registers through which field it was saved - so on edit we know how to build the options list.